### ISSUE TITLE
can user the biaxial coordinates to show variable with taget relationship

### ISSUE THREAD

**Issue by haiyuni** [fbdesignpro/sweetviz#73]
The distribution of independent variables and the distribution of independent variables with respect to the target variables are not in the same level in most cases. Double-coordinate axes can be used to display the distribution of independent variables and the distribution of independent variables to the target variables

**Comment by fbdesignpro** [comment]
Hello @haiyuni! I am guessing this is related to your other comment about a very low occurrence rate of the target. :) This one might be a bit more complicated but I will look into it and keep you posted here. Thank you again!

**Comment by haiyuni** [comment]
Well, in my model, the target ratio of the target variable is very low, which is not in the same range as the distribution of the variable. For example, the response rate of my target variable is 0.01%, but the distribution range of my variable will be between 10%-50%, so it cannot be displayed in one picture. I am looking forward to your reply,thanks!

**Comment by fbdesignpro** [comment]
@haiyuni yes that's what I figured; makes sense! From looking at it again, I remember thinking about this and chose to keep the single scale because it might get confusing, e.g. in the following example case, if there was a different scale at the bottom (let's say, from 0 to 30%), it would be very hard to tell which scale is applying to which data: 
![image](https://user-images.githubusercontent.com/65080980/107369323-a382a000-6aaf-11eb-8dc7-21176d9b0e5d.png)

I do want to keep Sweetviz as intuitive as possible for the majority of cases so I'm thinking of adding this as an option, so users would need to specify it to use it so they know what is going on. 

It would still need to be marked (e.g. adding a "Target %" label to the left and maybe some coloring), but I think it would work.

**Comment by haiyuni** [comment]
looking forward this packages

**Comment by fbdesignpro** [comment]
@haiyuni thank you for the ping, this should be part of the next version coming by the end of the month.

**Comment by fbdesignpro** [comment]
Just a quick update, it's still on my radar but my time has been taken by other projects as well as wrapping up 2.1.0. This will happen, it will just take a bit longer. Thank you for your patience!
